Code-4-Community/speak-for-the-trees-frontend
# Merge scaffold updates

## Why

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

There were a number of updates to the frontend-scaffold that had not yet been merged into this repo. This ensures everything is on the same page.

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers. 

fetched changes to the template and then went through and fixed a bunch of merge conflicts. Tried to be careful not to override anything SFTT specific

## Screenshots

Provide screenshots of any new components, styling changes, or pages. 

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves. 

We will have to see if stuff like coveralls and travis pass, the previous builds did not succeed
